app/src/main/java/com/hedvig/app/feature/home/ui/HomeFragment.kt

```diff
@@ -151,6 +151,10 @@ class HomeFragment : Fragment(R.layout.home_fragment) {
                 val items = mutableListOf<HomeModel>().apply {
                     addAll(listOfNotNull(*psaItems(successData.importantMessages).toTypedArray()))
                     add(HomeModel.BigText.Active(firstName))
+                    // TODO think about showing this in other states as well? Not just on isActive(...)
+                    if (successData.claimStatus.isNotEmpty()) {
+                        add(HomeModel.ClaimStatus(successData.claimStatus))
+                    }
```

Comment:
Need to check design-wise if these cards should also be shown when there isn't an active contract.
It feels like if someone has terminated their contract a day ago but they still have an ongoing claim from a week ago it should still be visible? Need to discuss this.

Reply:
I agree, but double-check with Hiro just to be safe.